src/lxc/tools/lxc_unshare.c

```diff
@@ -164,7 +164,7 @@ static bool lookup_user(const char *oparg, uid_t *uid)
 
 	if (sscanf(oparg, "%u", uid) < 1) {
 		/* not a uid -- perhaps a username */
-		if (sscanf(oparg, "%s", name) < 1) {
+		if (strlen(name) > PATH_MAX || sscanf(oparg, "%s", name) < 1) {
```

Comment:
Thanks.  This is good to do, however note that lxc-unshare is not setuid so it's not a privilege escalation.

However, shouldn't the comparison be >= PATH_MAX?

Reply:
yes, it should be `>=`, I've changed, please review agian